toolkits/global/packages/global-ratings-survey/view/ratingsSurvey.hbs

```diff
@@ -15,7 +15,7 @@
 				</div>
 				{{#if questionUrl}}
 					{{#if questionText}}
-						<a href="{{questionUrl}}?id={{id}}" target="_blank">{{questionText}} (opens in a new tab)</a>
+						<a href="{{questionUrl}}" data-ratings-survey="survey-link" target="_blank">{{questionText}} (opens in a new tab)</a>
```

Comment:
Links to external sites must have `rel="noopener"` [added for security reasons](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener) even though [browser support for implied `noopener` behaviour if opening links in new tabs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility) is pretty good.